scripts/patch-realsense-ubuntu-L4T.sh

```diff
@@ -91,7 +83,7 @@ case ${JETSON_L4T_VERSION} in
 		PATCHES_REV="5.0.2"	# JP 5.0.2
 		KERNEL_RELEASE="5.10"
 	;;
-	"36.3")
+	"36.3" | "36.4" | "36.4.3")
```

Comment:
Do we want to set 6.0 on 6.1 and 6.2 too?
Can we set 6.X maybe? and add a comment
```
36.3 --> 6.0
36.4 -> 6.1 
36.4.3 --> 6.2?
```

Reply:
OK that's great, can we explain it in the script like I suggested? or at least add a comment explaining why we continue with 6.0 even for 6.1 and 6.2 and the mapping?